packages/backend/src/logger.ts

```diff
@@ -58,6 +59,9 @@ const transports = [
         filename: 'logs/all.log',
         format: terminalFormat,
     }),
+    new LoggingWinston({
```

Comment:
is this going to error if not running in GCP or there's no GCP auth?

Reply:
I assumed it would be ignored since it recognises the project id when is running in their machines. 
But I assumed wrong and is throwing errors. It's fixed now.